### ISSUE TITLE
why not support python3

### ISSUE THREAD

**Issue by dywlegend** [alibaba/euler#4]
Good job!

but I have a question,why not support python3 ? python2 is outdated~

**Comment by YanZhangN** [comment]
Thank you for your interest，we will support python3 soon.

**Comment by renyi533** [comment]
Good catch. The python code is just thoroughly tested in python2 env. :(  We can commit to support python3 soon.

**Comment by mandyzore** [comment]
mark

**Comment by patricxu** [comment]
都鼠年了，出来吱一声啊

**Comment by usakey** [comment]
any progress on python 3 support?
